game/src/core/System.java

```diff
@@ -14,7 +14,7 @@
  * <p>A System needs to be registered with the Game via {@link Game#add(System)}.
  *
  * <p>This class is the abstract base class for each system. It implements the basic functionality
- * each system has. For example, it allows the system to be paused and unpaused.
+ * each system has. For example, it allows the system to be paused and unpause.
```

Comment:
Also entweder "it allows the system to pause and unpause" oder so wie es vorher war "it allows the system to be paused and unpaused".

Reply:
internet sagt "unpaused" gibt nit